conda-build/README.md

```diff
@@ -9,19 +10,19 @@ To then download the package, run ```conda install -c aihabitat -c conda-forge h
 
 ### Building for Linux
 
-The process is almost the same for linux; there is a corresponding python script for starting things off, however we use a docker container to do the builds. There is a dockerfile in this directory that you can use to create a container.
+The process is almost the same for linux; there is a corresponding python script for starting things off. You can use Ubuntu 16.04 or  a docker container to do the builds. There is a dockerfile in this directory that you can use to create a container.
```

Comment:
This is likely untrue unfortunately.  Using that old version of linux in the docker container is important to make the binaries be able to run on different platforms.  Maybe 16.04 is old enough to be the base these days, but we would need to do a lot of verification.

Reply:
Last binaries I built for v0.1.5 release were based on Ubuntu 16.04 that is used by our CI image.

Reply:
@mathfac That will work for most modern Linux releases, but may not work for certain older distributions. Particularly those that are stable for very long periods of time (CentOS for example). Older versions of Debian and Ubuntu may also not work.

Still probably best to build the official releases with that Docker for that reason.